ush/forecast_postdet.sh

```diff
@@ -545,22 +545,22 @@
 
   # Link CICE forecast output files from DATA/CICE_OUTPUT to COM
   local source_file dest_file
-  for fhr in ${CICE_OUTPUT_FH}; do
-    fhr3=$(printf %03i "${fhr}")
+  for fhr in ${CICE_OUTPUT_FH[*]}; do
```

Comment:
```suggestion
  for fhr in "${CICE_OUTPUT_FH[@]"}; do
```

Reply:
@WalterKolczynski-NOAA There were two things I needed to do to make these quotes work in postdet and use `mapfile` in predet:

1.  Add `fhr=$((10#${fhr}))` to line 549 in postdet to avoid invalid number errors when `fhr` has an 8. 
2.  Add `-d ' '` to `mapfile` in predet to make sure space is treated as a delimiter. 

I am testing and committing these changes. Hopefully this will resolve the spellcheck errors.